### PR TITLE
Fix require

### PR DESCRIPTION
**Pull request by marisaleung**
This change is [Reviewable](https://reviewable.io/reviews/google/blockly/1108)

### CODE REVIEW

File: core/workspace.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ goog.provide('Blockly.Workspace');
 
 goog.require('goog.array');
 goog.require('goog.math');
+goog.require('Blockly.VariableMap');
```
Comment (picklesrus) [inline core/workspace.js]: Nit: can you put this before goog.array so it is alphabetical?

Comment (marisaleung) [inline core/workspace.js]: Done

File: core/workspace.js [hunk 2]
```diff
@@ -24,6 +24,7 @@
  */
 'use strict';
 
+goog.require('Blockly.VariableMap');
```
Comment (picklesrus) [inline core/workspace.js]: doh. Sorry. The require should be after the provide... I think our eslint rules enforce this... but I'm not certain.

**Comment by RoboErikG** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[core/workspace.js, line 27 at r1](https://reviewable.io:443/reviews/google/blockly/1108#-KkDHnlZ929v5H3_NCia:-KkDHnlZ929v5H3_NCib:b84u9pp) ([raw file](https://github.com/google/blockly/blob/37d4a77607eff042c34d2caa9ccda61101cc44e3/core/workspace.js#L27)):*
> ```JavaScript
> 'use strict';
> 
> goog.require('Blockly.VariableMap');
> ```

Too far. After the provide (provide should always be above the requires)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/1108)*

**Comment by marisaleung** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[core/workspace.js, line 27 at r1](https://reviewable.io:443/reviews/google/blockly/1108#-KkDHiU7hcEt-q_x6skQ-r1-27:-KkDIMg0_NgqZW5Qzu5m:b-896fix) ([raw file](https://github.com/google/blockly/blob/37d4a77607eff042c34d2caa9ccda61101cc44e3/core/workspace.js#L27)):*
<details><summary><i>Previously, picklesrus wrote…</i></summary><blockquote>

doh. Sorry. The require should be after the provide... I think our eslint rules enforce this... but I'm not certain.
</blockquote></details>

Done.

---

*[core/workspace.js, line 27 at r1](https://reviewable.io:443/reviews/google/blockly/1108#-KkDHnlZ929v5H3_NCia:-KkDINBt3i1N5GN9Scq-:b-896fix) ([raw file](https://github.com/google/blockly/blob/37d4a77607eff042c34d2caa9ccda61101cc44e3/core/workspace.js#L27)):*
<details><summary><i>Previously, RoboErikG wrote…</i></summary><blockquote>

Too far. After the provide (provide should always be above the requires)
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly/1108)*

**Pull request merged**